src/hooks/useAnnouncementsUploads.tsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react'
+import { useClient } from 'cozy-client'
+import { AnnouncementsConfig } from 'components/Announcements/types'
+import flag from 'cozy-flags'
+
+const useAnnouncementsUploads = (url: string | undefined): string | null => {
+  const client = useClient()
+  const [image, setImage] = useState<string | null>(null)
+  const config = flag<AnnouncementsConfig>('home.announcements')
+
+  useEffect(() => {
+    const fetchImage = async (): Promise<void> => {
+      const urlWithoutPrefix = (url ?? '').replace('/uploads/', '')
+      try {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+        const fetchBinary = (await client?.stackClient.fetch(
+          'GET',
+          `/remote/${config.remoteDoctype}.uploads?url=${urlWithoutPrefix}`
+        )) as Response
+        const blob = await fetchBinary.blob()
+        setImage(URL.createObjectURL(blob))
```

Comment:
Can you explain why we need to fetch the image in JS side instead of giving the URL to the `<img>` element?

Reply:
I follow a similar process as for the wallpaper ([code](https://github.com/cozy/cozy-home/blob/3f0bf56705717c05962f28df854ddcd3c3068daf/src/hooks/useWallpaper.jsx#L6)) because I had a problem with authentification and CSP